Source/Core/Common/Crypto/SHA1.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <limits>
+#include <memory>
+#include <string_view>
+#include <type_traits>
+#include <vector>
+
+#include "Common/Assert.h"
+#include "Common/CommonTypes.h"
+
+namespace Common::SHA1
+{
+using Digest = std::array<u8, 160 / 8>;
```

Comment:
Was this 160 / 8 done intentionally?  Curious what the significance is.

Reply:
It’s somewhat common to talk about digest sizes as a bit length

Reply:
It might have been better to write `160 / CHAR_BIT` but that's not really that big of a deal (and `SHA1.h` was added by a separate PR, so it makes less sense to edit it in this one).

Reply:
well, SHA1 standard won't change if CHAR_BIT does :p